File: print-manager-dotnet-core-cs/ByteArrayExample.cs
using ceTe.DynamicPDF.Printing;

namespace print_manager_dotnet_core_cs
{
    class ByteArrayExample
    {
        public static void Run()
        {
            byte[] pdfBytes = System.IO.File.ReadAllBytes(Util.GetPath("Resources/DocumentA.pdf"));

            InputPdf pdf = new InputPdf(pdfBytes);
            PrintJob printJob = new PrintJob(Printer.Default, pdf);

            printJob.Print();
        }

    }
}

File: print-manager-dotnet-core-vb/ByteArrayExample.vb
Imports ceTe.DynamicPDF.Printing

Class ByteArrayExample
    Public Shared Sub Run()
        Dim pdfBytes As Byte() = System.IO.File.ReadAllBytes(Util.GetPath("Resources/DocumentA.pdf"))

        Dim pdf As New InputPdf(pdfBytes)
        Dim printJob As New PrintJob(Printer.Default, pdf)

        printJob.Print()
    End Sub
End Class
